docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgsql-postal.md

```diff
@@ -21,7 +21,35 @@ First install `libpostal` [from source](https://github.com/openvenues/libpostal)
 `make -j$(nproc) && sudo make install`
 ```
 
-To build `pgsql-postal` against the correct PostgreSQL version for YugabyteDB compatibility, install PostgreSQL 11 on your system as described in the [PostGIS example](../extension-postgis/).
+To build `pgsql-postal` against the correct PostgreSQL version for YugabyteDB compatibility, install PostgreSQL 11 on the your system using the following instructions applicable to your operating system.
+
```

Comment:
@sushantrmishra CentOS is deprecated. Do we need to provide instructions at all? ie just say "..., install PostgreSQL 11 on your system."